emerlin2caom2/main_app.py

```diff
@@ -71,59 +71,111 @@
 entry point that executes the workflow.
 """
 
-from os.path import basename
+import os
+import subprocess
 
-from caom2pipe import caom_composable as cc
-from caom2pipe import manage_composable as mc
+from caom2 import SimpleObservation, ObservationIntentType, Target, Telescope, TypedOrderedDict, Plane, Artifact, \
+    ReleaseType, ObservationWriter, ProductType, ChecksumURI
 
+import casa_reader as casa
+import measurement_set_metadata as msmd
```

Comment:
In casa_reader.py, msmd is casatools msmetadata, whereas here msmd is the locally created measurement_set_metadata.py; potential confusion for package readability.